### PR TITLE
Update domains.json

### PR DESCRIPTION
**Pull request by wwggyy2002**
Add Space ushiled

**If you want to create a space follow this guide**

https://docs.snapshot.page/guides/create-a-space

### CODE REVIEW

**Review by gokaiorg — APPROVED** [review]

**Comment by gokaiorg** [comment]
@wwggyy2002 Did you already create your ushiled.eth space, i cannot find it?

**Comment by wwggyy2002** [comment]
Sorry,my mistake,I write wrong space name.

**Comment by gokaiorg** [comment]
@wwggyy2002 Could you send me a link to your space? I still can't find `ltcapital.eth` space.

File: spaces/domains.json [hunk 1]
```diff
@@ -58,7 +58,8 @@
   "signal.idle.finance": "idlefinance.eth",
   "gov.flush.lol": "flushlol.eth",
   "fund.emptyset.finance": "esd.eth",
-  "mtrg.meter.io": "meter-mainnet.eth",
+   "mtrg.meter.io": "meter-mainnet.eth",
```
Comment (gokaiorg) [inline spaces/domains.json:61]: Beware of side effects.
```suggestion
  "mtrg.meter.io": "meter-mainnet.eth",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by wwggyy2002** [comment]
here is our space
https://snapshot.page/#/ltcapital.eth

**Comment by wwggyy2002** [comment]
> @wwggyy2002 Could you send me a link to your space? I still can't find `ltcapital.eth` space.

https://snapshot.page/#/ltcapital.eth

**Review by gokaiorg — CHANGES_REQUESTED** [review]
There is a conflict in your PR, can you resolve it?

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**